internal/plugins/helm/v1/init.go

```diff
@@ -128,19 +132,30 @@ func (p *initPlugin) InjectConfig(c *config.Config) {
 
 // Run will call the plugin actions
 func (p *initPlugin) Run() error {
-	return cmdutil.Run(p)
+	if err := cmdutil.Run(p); err != nil {
+		return err
+	}
+
+	// Update plugin config section with this plugin's configuration.
+	if err := p.config.EncodePluginConfig(pluginConfigKey, p.pluginConfig); err != nil {
+		return fmt.Errorf("error writing plugin config for %s: %v", pluginConfigKey, err)
+	}
+
+	return nil
 }
 
 // Validate perform the required validations for this plugin
 func (p *initPlugin) Validate() error {
 	// Check if the project name is a valid namespace according to k8s
-	dir, err := os.Getwd()
-	if err != nil {
-		return fmt.Errorf("error to get the current path: %v", err)
+	if p.pluginConfig.ProjectName == "" {
+		dir, err := os.Getwd()
+		if err != nil {
+			return fmt.Errorf("error to get the current path: %v", err)
+		}
+		p.pluginConfig.ProjectName = strings.ToLower(filepath.Base(dir))
```

Comment:
We should use `ProjectName` anywhere in the scaffolds that `os.Getwd` is used as a standin for the project name. Right now, I think that's in:
- `scaffolds/templates/kustomize.go`
- `scaffolds/templates/manager/config.go`